acceptance/analyzer_test.go

```diff
@@ -605,6 +601,35 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 					h.AssertPathExists(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.sha"))
 					h.AssertPathExists(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.toml"))
 				})
+
+				when("CNB_REGISTRY_AUTH is not provided", func() {
+					when("DOCKER_CONFIG is set", func() {
```

Comment:
The `CNB_REGISTRY_AUTH is not provided` context feel strange given that we don't have a `CNB_REGISTRY_AUTH is provided` context. Maybe we can get rid of the wrapping "not provided" context and change this `when` and the `when` of the above test to describe how credentials are provided.

Maybe something like... `when("registry creds are provided in the docker config.json"` vs `when("registry creds  are provided in CNB_REGISTRY_AUTH"`?